tests/Mock/TestTransportMock.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Requests\Tests\Mock;
+
+use Requests_Transport;
+
+class TestTransportMock implements Requests_Transport {
```

Comment:
```suggestion
class TransportTestMock implements Requests_Transport {
```
(class/interface name - method name - mock)

Reply:
It's supposed to be a type of transport, not a method name.
We already have these two in the code:
- `RawTransportMock`
- `TransportMock`